discord/abc.py

```diff
@@ -52,8 +53,6 @@
 T = TypeVar('T', bound=VoiceProtocol)
 
 if TYPE_CHECKING:
-    from datetime import datetime
```

Comment:
For this one I think `SnowflakeTime` can be moved inside this block instead. I'm not sure it's used anywhere outside of a type checking context anyway.

Reply:
Moved definition to inside the block.